languages/csharp/exercises/concept/dates/.meta/design.md

```diff
@@ -7,6 +7,7 @@ The goal of this exercise is to teach the student the basics of the Concept of D
 ## Learning objectives
 
 - Know of the existence of the `DateTime` type.
+- Know how to create a date.
```

Comment:
How to create a date => sign into Tinder 😜 
This is somewhat ambiguous, particularly given that different languages have different names for such types (`Date` or `DateTime` or both, or neither). Not sure if my suggestion below improves things though.

```suggestion
- Know how to create an object to hold date information.
```

Reply:
Agreed! I've rephrased to make it more explicit.